app.py

```diff
@@ -12,10 +12,13 @@ def create_app():
     def hello_world():
         return "Hello World"
 
+<<<<<<< HEAD
```

Comment:
I might be wrong but it looks like the merge conflict syntax is still present in your code. Can you remove it before I approve? I'm refering to the text that looks like this "<<<<<<<<< HEAD" or "========" Prof Breyden gets mad when these are left behind.

Reply:
![finn](https://github.com/user-attachments/assets/475ff443-6dfd-4a4d-986d-f3ca65b3b1a2)